Support/support_center_controller.rst

```diff
@@ -264,3 +264,35 @@ How can I use SAML for controller auth when I'm also using SAML for VPN authenti
 ------------------------------------------------------------------------------------------
 
 By default, we use "Hostname" for "Entity Id" when creating the SAML Endpoint in the Controller Console. When you create a second endpoint for controller login, you would have to pick "Custom" for "Entity Id" and use a custom string. You would have to use the same custom string for EntityId when you provision the SAML App at your IdP(Okta, Onelogin, Azure, etc)
+
+How can I restore my password when I lost or forget it ?
```

Comment:
Should we change title to 
How to reset Controller login password if it's lost or forgotten?